policy/modules/admin/alsa.fc

```diff
@@ -4,6 +4,7 @@ HOME_DIR/\.asoundrc				--	gen_context(system_u:object_r:alsa_home_t,s0)
 /etc/asound\.conf				--	gen_context(system_u:object_r:alsa_etc_t,s0)
 
 /run/alsa(/.*)?						gen_context(system_u:object_r:alsa_runtime_t,s0)
+/run/alsactl.pid				--	gen_context(system_u:object_r:alsa_runtime_t,s0)
```

Comment:
This dot needs to be escaped.
This is why Travis-CI test failed, https://travis-ci.org/SELinuxProject/refpolicy/jobs/596576535 contains:
```
/home/travis/build/SELinuxProject/refpolicy/policy/modules/admin/alsa.fc:7:
unescaped dot still present /run/alsactl.pid after being reduced to /run/alsactl.pid
(suggestion: use \. to match a dot, or a charset like [^/]))
```

Reply:
OK this is fixed now